tests/positive/storage/reformat_filesystem.go

```diff
@@ -49,7 +49,7 @@ func ReformatToBTRFS_2_0_0() types.Test {
 	        "format": "btrfs",
 	        "create": {
 	          "force": true,
-	          "options": [ "--label=OEM", "--uuid=CA7D7CCB-63ED-4C53-861C-1742536059CC" ]
+	          "options": [ "--label=OEM", "--uuid=CA7D7CCB-63ED-4C53-861C-1742536059D4" ]
```

Comment:
I'm assuming the UUID changes are to prevent collisions while running the tests parallel? Would it be worth it to just generate them?